### PR TITLE
Make the ResNet50 example compatible with TensorFlow 2.13

### PR DESCRIPTION
**Pull request by JanuszL**
## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- makes the ResNet50 example compatible with TensorFlow 2.13

### Affected modules and functionalities:
- TF RN50 example

### Key points relevant for the review:
- NA

### Tests:

- [x] Existing tests apply
  - TL3_RN50_convergence
  - TL1_tensorflow-dali_test
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py [hunk 1]
```diff
@@ -34,7 +38,10 @@ def __init__(self, rescaled_lr, step_boundaries, lr_values, warmup_steps,
     super(PiecewiseConstantDecayWithWarmup, self).__init__()
 
     self.rescaled_lr = rescaled_lr
-    self.step_boundaries = step_boundaries
+    if StrictVersion(tf.__version__) >= StrictVersion("2.13"):
+      self.step_boundaries = [int(b) for b in step_boundaries]
+    else:
+      self.step_boundaries = step_boundaries
```
Comment (stiepan) [inline docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py:44]: Can't we just stick to ints? Do the older versions require the number to be float?

Comment (JanuszL) [reply docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py:44]: Unfortunately it requires. See the previous build - relevant job failed with:
```
 ValueError: Boundaries (<dtype: 'int32'>) must have the same dtype as x (<dtype: 'float32'>).```

**Review by stiepan — APPROVED** [review]

File: docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py (hunk 1 above)
Comment (stiepan) [reply docs/examples/use_cases/tensorflow/resnet-n/nvutils/common.py:44]: Thanks!

**Review by szalpal — APPROVED** [review]

**Pull request merged**